tests/test_penalty.py

```diff
@@ -55,9 +57,13 @@ def prepare_test_ocp(with_muscles=False, with_contact=False, with_actuator=False
         nx = biorbd_model.nbQ() + biorbd_model.nbQdot()
         nu = biorbd_model.nbGeneralizedTorque()
     x_init = InitialGuess(np.zeros((nx, 1)))
-    u_init = InitialGuess(np.zeros((nu, 1)))
+    if implicit:
+        mod = 2
+    else:
+        mod = 1
```

Comment:
Please use one line if statement for these kind of small tests: 
`mod = 2 if implicit else 1`